src/Spectre.libWavelet.Tests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="gmock" version="1.7.0" targetFramework="native" />
+  <package id="Microsoft.Gsl" version="0.1.2.1" targetFramework="native" />
```

Comment:
We are still missing OpenCV in `packages.config`, to keep it in sync with `.vcxproj`.

Reply:
The test project will not need it, hence I'll remove the OpenCV references from .```vcxproj```.